geniza/corpus/models.py

```diff
@@ -621,6 +621,7 @@ def iiif_images(self, filter_side=False):
                         "image": img,
                         "label": labels[i],
                         "shelfmark": b.fragment.shelfmark,
+                        "not_in_document": b.side and i not in b.image_indices,
```

Comment:
the double negative in the template was making my head hurt a little bit, but I see now why you did it — it's easier to tell explicitly if an image is not part of a document / should be excluded, but included images could either be explicit or implicit

Reply:
maybe exclude or excluded would be clearer